test/node/features/express_disabled.feature

```diff
@@ -0,0 +1,62 @@
+Feature: @bugsnag/plugin-express autoDetectErrors=false
+
+Background:
+  Given I store the api key in the environment variable "BUGSNAG_API_KEY"
+  And I store the notify endpoint in the environment variable "BUGSNAG_NOTIFY_ENDPOINT"
+  And I store the sessions endpoint in the environment variable "BUGSNAG_SESSIONS_ENDPOINT"
+  And I start the service "express-disabled"
+  And I wait for the host "express-disabled" to open port "80"
+
+Scenario: a synchronous thrown error in a route
+  Then I open the URL "http://express-disabled/sync"
+  And I wait for 5 seconds
```

Comment:
I don't think you need to wait explicitly anymore; MR has a 30 second wait to check it doesn't get any requests already — https://github.com/bugsnag/maze-runner/blob/v4.11.3/lib/maze/configuration.rb#L9

Reply:
ah cool, I thought they were a bit slow!